packages/esm-patient-lists-app/src/workspaces/patient-lists.workspace.tsx

```diff
@@ -34,8 +38,9 @@ function PatientListsWorkspace() {
   const { patientLists, isLoading } = usePatientLists();
 
   const launchListDetailsWorkspace = useCallback((list) => {
-    closeWorkspace('patient-lists', true);
-    launchPatientWorkspace('patient-list-details', { list, workspaceTitle: list.name });
+    closeWorkspace({
+      onWorkspaceClose: () => launchPatientWorkspace('patient-list-details', { list, workspaceTitle: list.name }),
+    });
```

Comment:
I thought `patient-lists` and `patient-list-details` have the same type and therefore `patient-lists` would automatically get closed?

Reply:
They have the same type, so yes, this can be handled by the order thing, I'll get on it.
Thanks for the catch!